loop_live.py

```
@@ -0,0 +1,644 @@
+from utils.agent_display import AgentDisplay
+from utils.output_manager import OutputManager
+load_dotenv()
+install()
+
+ICECREAM_OUTPUT_FILE = Path.cwd() / "debug_log.json"
+# JOURNAL_FILE = Path.cwd() / "journal" / "journal.log"
+# JOURNAL_ARCHIVE_FILE = Path.cwd() / "journal" / "journal.log.archive"
+
+# JOURNAL_SYSTEM_PROMPT_FILE = Path.cwd() / "journal" / "journal_system_prompt.md"
+# PROMPT_NAME = None
+with open(SYSTEM_PROMPT_FILE, 'r', encoding="utf-8") as f:
```

Ruff (0.8.2)

`SYSTEM_PROMPT_FILE` may be undefined, or defined from star imports

(F405)